src/client/components/EditorWYSIWYG/hooks/_processCustomPaste.ts

```diff
@@ -0,0 +1,45 @@
+import serialize from 'dom-serializer'
+import { DomUtils, parseDocument } from 'htmlparser2'
+import type { IJodit } from 'jodit/esm/types/jodit'
+
+import { tableTags } from './_sanitizer'
+
+export const _processCustomPaste = (_: IJodit, html: string): string => {
+  const dom = parseDocument(html, { lowerCaseAttributeNames: true, lowerCaseTags: true })
+
+  const styledTableNodes = DomUtils.findAll((node) => {
+    return node.type === 'tag' && tableTags.includes(node.name) && typeof node.attribs.style === 'string'
+  }, dom.children)
+
+  const lengthRE = /^\d+(?:\.\d+)?(?:px|em|rem|%)$/
```

Comment:
Seems a bit strict, better any number of numbers (with dot), followed by any number of letters?